cpu/gd32v/periph/flashpage.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2020 Inria
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_stm32
+ * @{
+ *
+ * @file
+ * @brief       Low-level flash lock/unlock implementation
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ * @author      Oleg Artamonov <oleg@unwds.com>
+ * @author      Koen Zandberg <koen@bergzand.net>
+ *
+ * @}
+ */
+
+#include "cpu.h"
+#include "periph_cpu.h"
+#include "periph/flashpage.h"
+
+#define ENABLE_DEBUG        (0)
```

Comment:
```suggestion
#define ENABLE_DEBUG        0
```

With parenthesis, it is incompatible with `IS_ACTIVE()`

Reply:
Fixed